apps/o2-blocks/webpack.config.js

```diff
@@ -18,7 +22,11 @@ function getWebpackConfig( env, argv ) {
 			...webpackConfig.output,
 			filename: '[name].min.js',
 		},
-		plugins: [ ...webpackConfig.plugins, new ReadableJsAssetsWebpackPlugin() ],
+		plugins: [
+			...webpackConfig.plugins,
+			BuildMetaPlugin( { outputPath } ),
```

Comment:
should we base this on `webpackConfig.output` instead of having a separate variable? (same for happy-blocks)